prometheus.yml

```
@@ -5,12 +5,12 @@ scrape_configs:
   - job_name: 'gateway-service'
     metrics_path: /actuator/prometheus
     static_configs:
-      - targets: ['host.docker.internal:8080']
+      - targets: [ 'host.docker.internal:8080' ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)